fsspec/callbacks.py

```diff
@@ -58,8 +58,8 @@ def branched(self, path_1, path_2, kwargs):
             Child's source path
         path_2: str
             Child's destination path
-        kwargs: dict
-            arguments passed to child method, e.g., put_file.
+        **kwargs:
+            Arbitrary keyword arguments
```

Comment:
Before, this could be used to influence kwargs passed to `put_file`/`get_file`. I am not sure what the use case for that was, except of adding `callback` kwarg.

So this is deviating with `branch` in that regard.

Reply:
So you vote to remove and add back later if there is a use case?

Reply:
I don't see a use case for mutating kwargs in `branched()`. So, I'd just go with `**kwargs` (i.e. this PR).

I don't have any strong opinion against `kwargs=None` as it is similar but less ergonomic. But it does provide a way of mutating `kwargs` (but I don't see a use case for it).

We could also drop the third argument entirely, but that is not very flexible for future changes (I know I started with that, but 🤷🏽).